.github/workflows/test.yml

```diff
@@ -62,45 +57,60 @@ jobs:
         with:
           node-version: 12
 
-      - uses: actions/cache@v1
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-test-npm-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-test-npm-
-
-      - uses: actions/cache@v1
-        with:
-          path: ~/.cache
-          key: ${{ runner.os }}-test-cache-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-test-cache-
-
-      - name: Install Packages
-        run: |
-          npm i -g wait-on
-          npm ci
-          cd e2e
-          npm ci
+      - name: Install root dependencies
+        uses: bahmutov/npm-install@v1
 
       - name: Test Jest
         run: |
           npm test
 
-      - name: Test E2E mocked
-        run: |
-          npm run ng -- serve &
-          wait-on http://localhost:4200
-          cd e2e
-          node cypress-ci-e2e **/*mock*.e2e-spec.ts
-        env:
-          CI: true
-          PWA_BASE_URL: http://localhost:4200
-          ICM_BASE_URL: http://localhost:4200
-
-      - name: Upload results as an artifact
+  Cypress:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          [
+            '*mock*',
+            account/*b2b*,
+            quoting/*b2b*,
+            shopping/*b2b*,
+            account/*b2c*,
+            checkout/*b2c*,
+            contact/*b2c*,
+            shopping/*b2c*,
```

Comment:
this is an idea of partitioning, however if a new 'topic' is added, maybe it will get lost (theoretically speking) How about just having b2b and b2c here?

Reply:
is possible, just do it